mimoVision/vision.py

```diff
@@ -22,43 +21,73 @@ def get_vision_obs(self):
 class SimpleVision(Vision):
     def __init__(self, env, camera_parameters):
         super().__init__(env, camera_parameters)
+        self.viewer = None
+        self._viewers = {}
 
-        self.not_on_mac = sys.platform != 'darwin'  # Hacky check to try and intercept GLEW errors
-        self.offscreen_context = None
-        if self.not_on_mac:
-            self.offscreen_context = GlfwContext(offscreen=True)
+        if sys.platform == "win32":

```

Comment:
This doesn't work in Linux, but changing it to `if sys.platform != "darwin":` fixes it. @Domattee